src/main/java/frc/robot/Robot.java

```diff
@@ -485,6 +490,44 @@ public void robotPeriodic() {
     Logger.recordOutput(
         "AutoAim/Actual Distance",
         swerve.getPose().minus(FieldConstants.getSpeaker()).getTranslation().getNorm());
+
+    Tracer.endTrace();
+  }
+
+  @Override
+  public void loopFunc() {
+    Tracer.startTrace("Robot");
+    Tracer.traceFunc("LoopFunc", super::loopFunc);
```

Comment:
`traceFunc` exists specifically so you don't have to wrap everything in startTrace/endTrace.  Cut both of those here and everywhere else you're using traceFunc. There's enough of those that I'm not going to go through and annotate each unless you ask for it.